test/fixtures/get-orderbook-response.json

```diff
@@ -324,9 +324,9 @@
           "value": "0.5",
           "counterparty": "rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B"
         },
-        "totalPrice": {
+        "price": {
```

Comment:
Basically:

(desired) input divideBy quality == output
(desired) output multiply quality == input

There's not a term for `quality` that neatly expresses the above in one
word?

(Though I agree exchangeRate would be easily recognised)